configure.ac

```diff
@@ -247,7 +254,7 @@ AC_LANG_PUSH([C++])
 AX_CHECK_COMPILE_FLAG([-Werror],[CXXFLAG_WERROR="-Werror"],[CXXFLAG_WERROR=""])
 
 if test "x$enable_debug" = xyes; then
-    CPPFLAGS="$CPPFLAGS -DDEBUG -DDEBUG_LOCKORDER -DRSM_DEBUG_ASSERTION -DDEBUG_ASSERTION -DDEBUG_PAUSE"
+    CPPFLAGS="$CPPFLAGS -DDEBUG -DDEBUG_LOCKORDER -DRSM_DEBUG_ASSERTION -DDEBUG_ASSERTION -DDEBUG_PAUSE -D_GLIBCXX_DEBUG -DGLIB_CXX_DEBUG_PEDANTIC"
```

Comment:
this change means that `--enable-debug' implies `--enable-std-debug`. 

Is this done on purpose?

I mean what if I wanted to just run with `-D_GLIBCXX_DEBUG -DGLIB_CXX_DEBUG_PEDANTIC` and w/o `-DDEBUG_LOCKORDER -DRSM_DEBUG_ASSERTION -DDEBUG_ASSERTION -DDEBUG_PAUSE` ?

Reply:
nope, enable-debug is not supposed to have these flags, because you have to compile boost with these flags as well to use them.  I fixed